canvas_course_info/settings/base.py

```diff
@@ -35,17 +32,16 @@
     'django.contrib.staticfiles',
     'course_info',
     'icommons_ui',
-)
+]
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = [
```

Comment:
I see why the `ALLOWED_CIDR_NETS` setting isn't having an effect -- this needs to be added to the top of `MIDDLEWARE`: 
```
    'allow_cidr.middleware.AllowCIDRMiddleware',
```